### PR TITLE
Feat: campaigns view

### PR DESCRIPTION
**Pull request by schmanu**
## What this feature is about
We want to display ongoing and past campaigns to the user. Therefore we adjust the NavTabs component to offer all available campaigns sorted by most recent.

This PR also adjusts the campaign view and makes a few design tweaks.

## Screenshots
Screenshot 2024-06-25 at 16 05 13
Screenshot 2024-06-25 at 16 05 05

### CODE REVIEW

File: src/components/Points/index.tsx [hunk 1]
```diff
@@ -96,14 +82,13 @@ const Points = () => {
                     <Typography variant="h6" fontWeight={700} fontSize="20px">
                       {isGlobalCampaign ? 'Global' : campaign?.name}
                     </Typography>
+                    {isGlobalCampaign && (
+                      <Typography variant="body2" color="text.secondary">
+                        At this view you can see a total of points from all currently active campaigns and Regular Safe
```
Comment (usame-algan) [inline src/components/Points/index.tsx:87]:
```suggestion
                        In this view you can see the points total from all currently active campaigns and regular Safe
```

File: src/components/Points/index.tsx [hunk 2]
```diff
@@ -96,14 +82,13 @@ const Points = () => {
...
+                        Activities.
```
Comment (usame-algan) [inline src/components/Points/index.tsx:88]:
```suggestion
                        activities.
```

File: src/components/Points/CampaignLeaderboard.tsx [hunk 3]
```diff
@@ -243,14 +244,35 @@ export const CampaignLeaderboard = ({ campaign }: { campaign?: Campaign }) => {
...
           </Typography>
-        </Box>
-      </Box>
+          <Typography variant="h3" fontWeight={700} fontSize="32px">
+            {isGlobal ? 'Global' : campaign?.name} standings
+          </Typography>
+          <Typography variant="body2" color="text.secondary">
+            {isGlobal
+              ? 'In this leaderboard compete in overall standing in Partner Campaigns and Regular Safe activities.'
+              : 'See whats your standing in this campaign. Compete for the presents from the campaign partner.'}
```
Comment (usame-algan) [inline src/components/Points/CampaignLeaderboard.tsx:263]:
```suggestion
              : 'See whats your standing in this campaign. Compete for presents from the campaign partner.'}
```

File: src/components/Points/CampaignLeaderboard.tsx [hunk 4]
```diff
@@ -243,14 +244,35 @@ export const CampaignLeaderboard = ({ campaign }: { campaign?: Campaign }) => {
...
     <Box key={campaign?.resourceId}>
-      <Box sx={{ display: 'flex' }}>
-        <SvgIcon component={TitleStar} inheritViewBox sx={{ mr: '8px', mt: '4px' }} />
-        <Box sx={{ flex: '1' }}>
-          <Typography variant="h2" fontWeight={700} sx={{ mr: '8px', display: 'inline' }}>
-            {isGlobal ? 'Global' : campaign?.name} leaderboard
+      <Stack direction="row" width="100%" justifyContent="space-between">
+        <Stack spacing={1}>
+          <Typography
+            sx={{ fontFeatureSettings: `'clig' off, 'liga' off` }}
```
Comment (usame-algan) [inline src/components/Points/CampaignLeaderboard.tsx:250]: This is interesting. What does it do?

**Review by usame-algan — APPROVED** [review]
Just some minor text suggestions. Looks good overall 👍

File: src/components/Points/CampaignTabs.tsx [hunk 5]
```diff
@@ -1,65 +1,110 @@
...
+}
+
+const CampaignTabs = ({
+  onChange,
+  selectedCampaignId,
+}: {
+  onChange: (campaignId: string) => void
+  selectedCampaignId: string
+}) => {
+  const campaigns = useCampaignPage(5)
```
Comment (usame-algan) [inline src/components/Points/CampaignTabs.tsx:45]: Are we ordering the campaigns in any way e.g. newest on top?

Comment (schmanu) [reply src/components/Points/CampaignTabs.tsx]: Currently its not possible to sort them somehow. But it might be added to the backend.

**Comment by francovenica** [comment]
Small visual issue.
According to the desings, as you select campaings the green line in the border should show. Currenlty the line only shows if you select "Global" and not for the campaigns

Green line

The rest looks good to me

**Comment by francovenica** [comment]
The Green line looks fine.

I noticed something that I'm pretty sure didn't happened before. The "Load more" at the bottom shows even when there are no more campaigns to load. Did something change in the pagination?

Load more

**Comment by liliya-soroka** [comment]
Verified

**Pull request merged**